prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -17,6 +19,14 @@ asap_accepted_issuers = { "{{ join "\",\"" (splitList "," .Env.JWT_ACCEPTED_ISSU
 asap_accepted_audiences = { "{{ join "\",\"" (splitList "," .Env.JWT_ACCEPTED_AUDIENCES) }}" }
 {{ end }}
 
+{{ if $ENABLE_XMPP_WEBSOCKET }}
+  {{ if .Env.PUBLIC_URL }}
```

Comment:
There should be no else here, this variable should be required for WS to work.

Reply:
Done + Add a check in 10-config